.releaserc

```diff
@@ -4,7 +4,7 @@
     {
       "//": "build the alpine, macos, linux and windows binaries",
       "path": "@semantic-release/exec",
-      "cmd": "npm i -g pkg && pkg . -t node14-alpine-x64,node14-linux-x64,node14-macos-x64,node14-win-x64"
+      "cmd": "npm i -g pkg && pkg . --options max_old_space_size=16384 -t node14-alpine-x64,node14-linux-x64,node14-macos-x64,node14-win-x64"
```

Comment:
Suggestion: I think we should pin the version and not use latest all the time.

Reply:
Which version do you mean? For the `pkg` tool?

Reply:
yes, `npm i -g pkg` installs the latest version of pkg

Reply:
Could you share more about the benefits you see in pinning, and what issues we're aiming to prevent?

Pinning does bring build stability, but considering that `pkg` is just a build tool and our pipeline doesn't run often as far as I can see, I'm more concerned about being outdated over time and missing bug fixes or improvements of the tool.

Reply:
* Currently also major upgrades will be included even breaking changes, which is problematic.
* PKG is not just a build tool, it determines which node version is being bundled in the binaries.
* Pinning doesn't mean to not consume bugfix releases, but at least not automatically consume major changes.